update time should be updated

josefree opened the pull request:
1. update time should be updated accordingly
2. for Mooncake new services, better to go to what's new page.

azurecla commented:
Hi __@v-cmans__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-cmans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

josefree committed:
update time should be updated

1. update time should be updated accordingly
2. for Mooncake new services, better to go to what's new page.
